test/ttmlir/Silicon/TTNN/unsqueeze.mlir

```diff
@@ -0,0 +1,102 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path%" %s  > %t.mlir
+// RUN: FileCheck %s --input-file=%t.mlir
+// RUN: ttmlir-translate --ttnn-to-flatbuffer %t.mlir > %t.ttnn
+func.func @neg_dim_five(%arg0: tensor<4x2x32x32xbf16>) -> tensor<1x4x2x32x32xbf16> {
+  %0 = tensor.empty() : tensor<1x4x2x32x32xbf16>
+  // CHECK: %[[C:.*]] = "ttnn.reshape"
```

Comment:
Replace `%[[C:.*]]` with `%{{.*}}`, as `C` is not used in the rest of the function, or remove it completely.